_implement/scl-windows.md

```diff
@@ -63,8 +57,8 @@ Check the following items **before** reviewing these network guides and lessons
 
 1. Users have PIV credentials and PIV card readers.
 1. You are using Microsoft Active Directory to manage your Windows network.
-1. Domain Controllers are Microsoft 2012 or newer.
-1. User workstations **are joined** to your network and are Windows 8 or Windows 10-based.
+1. Domain Controllers are running Microsoft Windows Server 2019 or newer.
+1. User workstations **are joined** to your network and are Windows 8 or newer.
```

Comment:
Windows 8 is no longer supported. We should specify Windows 10 or later.

Reply:
agreed